packages/cli/src/error_handler.ts

```diff
@@ -107,16 +110,21 @@ const handleError = async (
     if (errorHasCauseMessage(error)) {
       printer.print(`Cause: ${error.cause.message}`);
     }
-    await usageDataEmitter?.emitFailure(error, { command: command ?? '' });
+    await usageDataEmitter?.emitFailure(error, {
+      command: command ?? 'UnknownCommand',
```

Comment:
maybe the default should be `InvalidCommand` instead? Because I think the only way it would be undefined is if we failed to parse it which means it was invalid. Unknown makes it seems like it might have been valid we just didn't have logic to parse it.

Reply:
`handleError` is also used to handle uncaught and unhandle exception at the process level. The command could be undefined when something unexpected happens.